src/BioentityProperty.js

```diff
@@ -1,100 +1,90 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-class BioentityProperty extends React.Component {
 
-    constructor(props) {
-        super(props)
-        this.state = {
-            showAll: false
-        }
+const PropertyValue = ({hasUrl, isLast, text, url}) =>
+  <span>
+    {hasUrl &&
+      <a className={"bioEntityCardLink"} href={url} target="_blank">{text}</a>
     }
 
-    // take three most relevant links and then all of the same relevance
-    _pickMostRelevant(properties) {
-        const relevanceThreshold =
-            properties
-                .map(p => p.relevance)
-                .sort((l,r) => r-l)
-                .concat([0,0,0])
-                [properties.size < 3 ? properties.size-1 : 2];
-        return properties.filter(p => p.relevance>=relevanceThreshold);
+    {!hasUrl &&
+      <span>{text}</span>
     }
```

Comment:
Consider the [ternary operator idiom](https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator) instead of two exclusive conditional render blocks.